lpc845-test-stand/test-assistant/src/handle_msg.rs

```diff
@@ -125,12 +134,13 @@ pub fn handle_idle(cx: crate::idle::Context) -> ! {
                                 }
                             }
                             CTS_PIN_NUMBER => {
-                                if cts.direction_is_output() {
-                                    match level {
-                                        pin::Level::High => cts.set_high(),
-                                        pin::Level::Low => cts.set_low(),
-                                    }
-                                }
+                                todo!("AJM: Refactor")
```

Comment:
This should probably be resolved (check fixed direction pin list?) before merging.